### ISSUE TITLE
Syntax error from pip 1.1.7

### ISSUE THREAD

**Issue by bevenky** [fritzy/SleekXMPP#178]
Installing collected packages: sleekxmpp
  Running setup.py install for sleekxmpp
    SyntaxError: ('invalid syntax', ('/Users//src/python_env/lib/python2.7/site-packages/sleekxmpp/plugins/xep_0065/stream.py', 17, 21, '        host = self.\n'))

```
SyntaxError: ('invalid syntax', ('/Users//src/python_env/lib/python2.7/site-packages/sleekxmpp/xmlstream/tcp_binding.py', 1, 20, 'connection discovery:\n'))
```

**Comment by legastero** [comment]
Looks like setup.py picked up a few too many files when I built the package. Thanks!

**Comment by bevenky** [comment]
Kewl. Can we update that. :) Looking to trying it out.
*
*

On Sat, Jun 30, 2012 at 5:37 PM, Lance Stout <
reply@reply.github.com

**Comment by legastero** [comment]
That should be fixed now, and with 1.1.8.

**Comment by bevenky** [comment]
Is 1.1.8 up on pip now?
*
*
Regards

_Venky, *Co-Founder at Plivo Inc
*cell: *567-703-6815
*skype:_ venkateshb11
_web:_ www.plivo.com | twitter: @plivo http://www.twitter.com/plivo

On Sat, Jun 30, 2012 at 5:45 PM, Lance Stout <
reply@reply.github.com
